data api/src/main/java/de/tinf15b4/kino/data/ServiceImplModel.java

```diff
@@ -53,24 +52,24 @@ public E findOne(long id) {
 
         Optional<E> oe = save(e);
         if (oe.isPresent()) {
-            e = oe.get();
+            E updated = oe.get();
 
-            long newid = e.getId();
+            long newid = updated.getId();
 
             if (newid != oldid && oldid != 0) {
-                String table = e.getClass().getAnnotation(Table.class).name();
+                String table = updated.getClass().getAnnotation(Table.class).name();
 
                 // detach old entity
                 entityManager.flush();
-                entityManager.detach(e);
+                entityManager.detach(updated);
 
                 // adjust the id
-                entityManager.createNativeQuery("UPDATE " + table + " SET id = " + oldid + " WHERE id = " + newid).executeUpdate();
+                entityManager.createNativeQuery(String.format("UPDATE %s SET id = %s WHERE id = %s", table, oldid, newid)).executeUpdate();
```

Comment:
Sure the intention was to use the predefined query thingy. I just could figure out how to do it in this context ^^
I found no method that allowed to do something like taht. Probably should have checked in more details here